src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -361,79 +362,85 @@ function AvatarCropModal(props) {
             type={CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED}
             onModalHide={resetState}
         >
-            {props.isSmallScreenWidth && <HeaderGap />}
-            <HeaderWithBackButton
-                title={props.translate('avatarCropModal.title')}
-                onBackButtonPress={props.onClose}
-            />
-            <Text style={[styles.mh5]}>{props.translate('avatarCropModal.description')}</Text>
-            <GestureHandlerRootView
-                onLayout={initializeImageContainer}
-                style={[styles.alignSelfStretch, styles.m5, styles.flex1, styles.alignItemsCenter]}
+            <ScreenWrapper
```

Comment:
@hungvu193, can we just wrap [BaseModal](https://github.com/Expensify/App/blob/main/src/components/Modal/BaseModal.js) with the screen wrapper?

Reply:
> The best way to do that is wrapping ScreenWrapper with any modal that's full screen modal

Is there a way we could detect that? The current approach is also fine, but I'm wondering if we could have a more universal solution